actionview/lib/action_view/helpers/csrf_helper.rb

```diff
@@ -21,10 +21,19 @@ module CsrfHelper
       #
       def csrf_meta_tags
         if defined?(protect_against_forgery?) && protect_against_forgery?
-          [
-            tag("meta", name: "csrf-param", content: request_forgery_protection_token),
-            tag("meta", name: "csrf-token", content: form_authenticity_token)
-          ].join("\n").html_safe
+          if session.respond_to?(:enabled?) && !session.enabled?
+            if Rails.application.config.action_dispatch.silence_disabled_session_errors
```

Comment:
I don't like this `Rails.application.config` reach, but I'm unsure what a better alternative would be.

Reply:
Maybe the config would be better defined on `action_controller` so we can add it to `CONTROLLER_DELEGATES` like how we handle `request_forgery_protection_token`? We could also add a `session_enabled?` or `session_disabled?` helper method similar to `protect_from_forgery?`.

Reply:
Oh, actually that makes me think we could do this check in `protect_against_forgery?`. no?

Reply:
Yes, that probably makes sense.